lib/App/MarathonDeployer.pm

```diff
@@ -69,6 +76,35 @@ sub BUILD {
 
         $self->marathon_json->{instances} = int($self->marathon_instances);
     }
+
+    die 'CPU_PROFILE ' . $self->cpu_profile . ' is not one of low|standard|high'
```

Comment:
Please document the `CPU_PROFILE` option and its acceptable values in README.md..